src/modules/user/dto/user-profile.input.ts

```diff
@@ -0,0 +1,5 @@
+export class UserProfileInput {
```

Comment:
Do we still need this @philip-calape @arnellebalane ?
There is already a `UserProfile` class that always mirrors the `UserProfileInput`.

Reply:
Hello @japhethbalane yes we need to introduce a separate class for inputs the same with outputs in order to introduce a layer of separation between our models passed on our server. For now the input, output, and model classes are exactly the same but if ever we add another functionality which requires different input/output from our model to be used by our server, this separation will be helpful. Thanks!